Sources/SwiftSyntax/Syntax.swift

```diff
@@ -129,6 +132,7 @@ public struct Syntax: SyntaxProtocol, SyntaxHashable {
   }
 
   /// "designated" memberwise initializer of `Syntax`.
+  @_transparent
```

Comment:
Do we need `@_transparent` here? It does make debugging odd because you can no longer step into the function. Isn‘t `@inline(__always)` enough? Same for the other `@_transparent` functions. https://github.com/swiftlang/swift/blob/main/docs/TransparentAttr.md for reference.

Reply:
Hmm, that’s unfortunate. Could you add a comment with this explanation to the `@_transparent` attribute?